maat-court-data-api/src/test/java/gov/uk/courtdata/prosecutionconcluded/helper/ReservationsRepositoryHelperTest.java

```diff
@@ -34,7 +32,7 @@ public void testWhenMaatIsNotLocked_thenReturnTrue() {
                 .build());
         when(reservationsRepository.findById(anyInt())).thenReturn(reservationsEntity);
 
-        boolean status = reservationsRepositoryHelper.isMaatRecordLocked(anyInt());
```

Comment:
I think it was being flagged by the SQ because of line 55 was checking for nulls. It makes sense what you're saying though as it's a primitive returned (I didn't check).